clients/admin-ui/src/features/config-wizard/config-wizard.slice.ts

```diff
@@ -4,30 +4,22 @@ import type { RootState } from "~/app/store";
 import { DEFAULT_ORGANIZATION_FIDES_KEY } from "~/features/organization";
 import { Organization, System } from "~/types/api";
 
-import { REVIEW_STEPS, STEPS } from "./constants";
+import { STEPS } from "./constants";
 import { AddSystemMethods, SystemMethods } from "./types";
 
 export interface State {
   step: number;
-  reviewStep: number;
   organization?: Organization;
   /**
    * The systems that were returned by a system scan, some of which the user will select for review.
    * These are persisted to the backend after the "Describe" step.
    */
   systemsForReview?: System[];
-  /**
-   * The system that is currently being put through the review steps. It is persisted
-   * on the "Describe" step and then updated with additional info. Once it's registered,
-   * the next `systemForReview` is swapped in, if any.
-   */
-  systemInReview?: System;
```

Comment:
this hasn't been doing anything for a while now, since we removed the reviewing systems one by one step towards the end of last year